### ISSUE TITLE
Request: informal communication channel

### ISSUE THREAD

**Issue by wimrijnders** [mn416/QPULib#49]
@mn416 I would appreciate a method of communicating without having to use the issues here. This is just to keep the issues here unpolluted with social chitchat.

As a suggestion, the [vis.js](http://visjs.org/)[1] project uses  a [Gitter Lobby](https://gitter.im/vis-js/Lobby) for informal communication. This works quite well, especially because it's coupled with `github`.

The idea is to have a place for questions, comments, small-talk etc. No direct response is expected (although you then *do* have the option for a live chat), you can totally respond in your own time.

Hope you find this appealing as well.

I just discovered that I can create a room on `gitter` myself. If I do so, will you join?

-----

[1] *Of which I have been an active maintainer for over a year - I'm still a maintainer officially but don't do work there any more.*

**Comment by wimrijnders** [comment]
I can set up the room, but I can't integrate with the `mn416/QPULib` project, because I'm not an admin. I'm hoping you can do that.

In any case [here is the QPULib lobby](https://gitter.im/QPULib/Lobby#).

**Comment by mn416** [comment]
By the way, `make test` has already proved useful, thanks :)

**Comment by wimrijnders** [comment]
>  I think it's important to keep discussions fairly concise and the issue tracker will force that.

Which actually is one reason for having a separate channel for 'other stuff', to not clutter the issues. But fine, I'll let it go.

> Hopefully in a few days I'll be ready to commit it.

Great! Happy to hear it. I've been busy with `Rot3DLib`. I realized that I want to put quite a lot of code in there, for testing and demonstration, so I put some time in making the handling of command line parameters decent. I took some previous code for this and continued from there.

However, that soon escalated to a point where it was becoming a project on itself, so I spun it off. I will put a 'lite' version in `Rot3DLib`[1].

-------
[1] *This is also a demonstration of the kind of chitchat I would prefer to avoid in issues.*
